src/style.css

```diff
@@ -0,0 +1,146 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+p {
+  margin: 0;
+}
+
+h2 {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  width: 200px;
+  padding: 32px 16px 16px;
+  flex-direction: column;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: flex;
+  margin: auto;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.card__id {
+  margin-bottom: 16px;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+/* #region stars */
+
+.stars {
+  display: flex;
+  margin: 0 ;
+}
+
+.star {
+  height: 16px;
+  width: 16px;
+  margin: 0 2px;
+
+  background-repeat: no-repeat;
+  background-image: url(./images/star.svg);
+  background-position: center;
+}
+
+.stars--1 .star:nth-child(1),
+.stars--2 .star:nth-child(-n+2),
+.stars--3 .star:nth-child(-n+3),
+.stars--4 .star:nth-child(-n+4),
+.stars--5 .star:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
+
+/* #endregion  */
+
+.card__feedback {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.feedback {
+  align-self: flex-end;
+
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.price {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.cost {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  width: 166px;
+  height: 40px;
+
+  background: #00acdc;
+  border-radius: 5px;
+}
+
+.buy {
+  width: 100%;
+
+  text-decoration: none;
+  text-align: center;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+}
+
+.card__buy--hover:hover {
+  background: #fff;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+}
+
+.buy--hover:hover {
+  color: #00acdc;
+}
```

Comment:
Button `купить` not all area can be pressed.
You can remove not necessary `div` around link.
.
Also you can centrate button horisontaly in case if container will change width

Reply:
you also could mention about BEM, because it's wrong here)